tls/s2n_x509_validator.c

```diff
@@ -732,6 +732,21 @@ S2N_RESULT s2n_validate_certificate_signature(struct s2n_connection *conn, X509
     const struct s2n_security_policy *security_policy;
     RESULT_GUARD_POSIX(s2n_connection_get_security_policy(conn, &security_policy));
 
+    /**
+     *= https://www.rfc-editor.org/rfc/rfc5246#section-7.4.2
```

Comment:
This probably needs a comment explaining why these compliance comments are here, since I'm not sure it's obvious.